### PR TITLE
the canister

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Petrel pumps now have a fancy canister
Petrel pumps now make fuel in 120 batches rather then 60
corrects grammar 

## Changelog
:cl:
/:cl:

### CODE REVIEW

File: code/game/machinery/kitchen/petrel_maker.dm [hunk 1]
```diff
@@ -2,19 +2,20 @@
 /obj/machinery/petrel_maker
 	name = "Greyson Petrel Pump"
```
Comment (ThePainkiller) [inline code/game/machinery/kitchen/petrel_maker.dm:3]:
```suggestion
	name = "Greyson Petrol Pump"
```
I want to believe you meant like, Petrol pump, as in, from a gas station. With an o.
Might need refactor to make it consistent

File: code/game/machinery/kitchen/petrel_maker.dm [hunk 2]
```diff
@@ -2,19 +2,20 @@
...
 	desc = "A machine for petrel and refined scrap. Originally design by the Greyson Positronic's AI then given to the Artificer's Guild. \
-	When stored pastic as the same or higher just flip the switch on its side to produce a bottle of diesel. \
+	When stored plastic as the same or higher just flip the switch on its side to produce a bottle of diesel. \
```
Comment (ThePainkiller) [inline code/game/machinery/kitchen/petrel_maker.dm:5]:
```suggestion
	When enough plastic is stored inside, flip the switch on its side to produce a bottle of diesel relative to the ammount of plastic used. \
```

File: code/game/machinery/kitchen/petrel_maker.dm [hunk 3]
```diff
@@ -2,19 +2,20 @@
 /obj/machinery/petrel_maker
 	name = "Greyson Petrel Pump"
 	desc = "A machine for petrel and refined scrap. Originally design by the Greyson Positronic's AI then given to the Artificer's Guild. \
```
Comment (ThePainkiller) [inline code/game/machinery/kitchen/petrel_maker.dm:4]:
```suggestion
	desc = "A machine for production of petrol and refined scrap. Originally designed by the Greyson Positronic's AI, then given to the Artificer's Guild. \
```

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 4]
```diff
@@ -121,9 +121,13 @@
 	preloaded_reagents = list("adminordrazine" = 60)
 
 /obj/item/weapon/reagent_containers/glass/bottle/petrel
-	name = "diesel bottle"
-	desc = "A small bottle. Contains the liquid diesel useful for powering diesel generators or fire bombs."
-	preloaded_reagents = list("fuel" = 60)
+	name = "diesel canister"
+	desc = "A small canister. Contains the liquid diesel useful for powering diesel generators or fire bombs."
```
Comment (ThePainkiller) [inline code/modules/reagents/reagent_containers/glass/bottle.dm:125]:
```suggestion
	desc = "A small canister. Contains liquid diesel, useful for powering diesel generators or making fire bombs."
```

**Review by ThePainkiller — CHANGES_REQUESTED** [review]
Lillyyyyyyyyy

**Comment by Trilbyspaceclone** [comment]
Petrel is petrel

**Pull request merged**